src/map/packet_system.cpp

```diff
@@ -1018,18 +1018,30 @@ void SmallPacket0x028(map_session_data_t* const PSession, CCharEntity* const PCh
     int32 quantity  = data.ref<uint8>(0x04);
     uint8 container = data.ref<uint8>(0x08);
     uint8 slotID    = data.ref<uint8>(0x09);
+    CItem* PItem    = PChar->getStorage(container)->GetItem(slotID);
+    uint16 ItemID   = PItem->getID();
 
     if (container >= MAX_CONTAINER_ID)
     {
         ShowExploit(CL_YELLOW "SmallPacket0x028: Invalid container ID passed to packet %u by %s\n" CL_RESET, container, PChar->GetName());
         return;
     }
 
-    CItem* PItem = PChar->getStorage(container)->GetItem(slotID);
+    if (ItemID > 29311 && ItemID < 29340)
+    {
+        int data = 0;
+        for (int i = 0; i < 0x18; i++)
+            data += PItem->m_extra[i];
+
+        if (data != 0)
+        {
+            PChar->pushPacket(new CMessageStandardPacket(MsgStd::CannotBeProcessed));
+            return;
+        }
```

Comment:
I was sending you snippets from my phone last night, this should be formatted as:

```cpp
int data = 0;
for (int i = 0; i < CItem::extra_size; i++)
{
    data += PItem->m_extra[i];
}

if (data != 0)
{
    PChar->pushPacket(new CMessageStandardPacket(MsgStd::CannotBeProcessed));
    return;
}
```